tools/install-deps.sh

```diff
@@ -20,7 +20,7 @@ rm -rf libsodium
 
 # install libopus, needed for audio encoding/decoding
 if ! [ -f $CACHE_DIR/usr/lib/pkgconfig/opus.pc ]; then
-  curl http://downloads.xiph.org/releases/opus/opus-1.1.tar.gz -o opus-1.1.tar.gz
+  curl https://downloads.xiph.org/releases/opus/opus-1.1.tar.gz -o opus-1.1.tar.gz
```

Comment:
this does not work, the result is a HTML page.

```
$ curl https://downloads.xiph.org/releases/opus/opus-1.1.tar.gz -o opus-1.1.tar.gz 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   433  100   433    0     0    583      0 --:--:-- --:--:-- --:--:--   582

$ cat opus-1.1.tar.gz 
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>302 Found</title>
</head><body>
<h1>Found</h1>
<p>The document has moved <a href="https://ftp.osuosl.org/pub/xiph/releases/opus/opus-1.1.tar.gz">here</a>.</p>
<hr>
<address>Apache/2.4.10 (Debian) SVN/1.8.10 mod_fcgid/2.3.9 mod_python/3.3.1 Python/2.7.9 OpenSSL/1.0.1t mod_perl/2.0.9dev Perl/v5.20.2 Server at downloads.xiph.org Port 443</address>
</body></html>
```

Reply:
Add the `-L` option to curl to make it follow redirects.

Reply:
Done.